scripts/publish.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# For local development, in production, the environment will be set though GH actions and GH secrets
+if [ -f ".envrc" ]; then
+    echo "Loading .envrc"
+    source .envrc
+else
+    echo "No .envrc"
+fi
+
+echo "============================"
+echo "PAT: $PAT"
+echo "PRODUCTION: $PRODUCTION"
+echo "============================"
+
+if [ "$PRODUCTION" = "1" ]; then
+    git config --global user.email "prismabots@gmail.com"
+    git config --global user.name "Prismo"
+    git remote add github "https://$GITHUB_ACTOR:$GH_TOKEN@github.com/$GITHUB_REPOSITORY.git" || true
+else
+    echo "Not setting up repo because PRODUCTION is not set"
+fi
+
+if [ -z "$PAT" ]; then
+    echo "\$PAT is empty. Please set the value of $PAT"
+elif [ -n "$PAT" ]; then
+    if [ "$PRODUCTION" = "1" ]; then
+        ./node_modules/.bin/vsce publish patch --pat $PAT
```

Comment:
Might be useful to add a quick comment here like "Publishing patch release $PAT".

Reply:
> Might be useful to add a quick comment here like "Publishing patch release $PAT".

Done without the `$PAT` part as it is a secret! ;)